sql/functions/pgbouncer_fdw_functions.sql

```diff
@@ -100,7 +101,50 @@ LOOP BEGIN
     INTO v_version_major, v_version_minor
     FROM @extschema@.pgbouncer_version_func(v_row.target_host);
   
-    IF v_version_major >= 1 AND v_version_minor >= 18 THEN 
+    IF v_version_major >= 1 AND v_version_minor >= 21 THEN 
```

Comment:
```suggestion
    IF v_version_major = 1 AND v_version_minor >= 21 THEN 
```

Reply:
Why change this to exactly equal to? If 2.0 came out and there were no changes to the column names (unlikely as that would be), it would work perfectly fine.

Reply:
Ok. But the logic should definitely apply to the in-between versions.

Reply:
Will apply logic to other sections. Thanks!